Makefile

```diff
@@ -114,10 +114,10 @@ test_race:
 	@go test -race $(PACKAGES_NOCLITEST)
 
 test_sim:
-	@echo "Running individual module simulations..."
+	@echo "Running individual module simulations."
 	@go test $(PACKAGES_SIMTEST) -v
-	@echo "Running full Gaia simulation..."
-	@ENABLE_GAIA_SIMULATION=1 go test ./cmd/gaia/app -run TestFullGaiaSimulation -v
+	@echo "Running full Gaia simulation. This may take several minutes. Set the environment variable 'GAIA_SIMULATION_SEED' to run with a constant seed."
```

Comment:
Instead of an environment variable, can this use a build flag? ref: https://github.com/cosmos/cosmos-sdk/issues/1665

Granted that proposal wasn't agreed on, so feel free to ignore this point. 
We can also save this for a future PR.

Reply:
Hmm, I think we want simulated tests on PRs to develop, the length is configurable (the LCD tests take longer right now anyways...), even 60s of testing is helpful.

Reply:
LCD tests should be ~45 seconds right?

But I agree. Can we add a time parameter that tells it how many blocks to simulate then?

Reply:
So added.